indico/modules/events/persons/__init__.py

```diff
@@ -0,0 +1,36 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2016 European Organization for Nuclear Research (CERN).
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 3 of the
+# License, or (at your option) any later version.
+#
+# Indico is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Indico; if not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import unicode_literals
+
+from flask import session
+
+from indico.core import signals
+from indico.util.i18n import _
+from indico.web.flask.util import url_for
+from indico.web.menu import SideMenuItem
+
+
+@signals.menu.items.connect_via('event-management-sidemenu')
+def _sidemenu_items(sender, event, **kwargs):
+    can_modify = event.can_manage(session.user, allow_key=True)
+    is_lecture = event.type == 'simple_event'
```

Comment:
i mean in case it's a lecture - you never show  the item in that case.

```
if event.type == 'lecture' or not event.can_manage(session.user, allow_key=True):
    return
```